### PR TITLE
[26.1] final changes for 26.1rc1

### PR DESCRIPTION
**Pull request by glozow**
Final changes to tag a 26.1rc1.
Bumps version numbers, man pages, adds release notes etc.

### CODE REVIEW

**Comment by hebasto** [comment]
To fix the "macOS 13 native" job failure we should backport https://github.com/bitcoin/bitcoin/pull/29195.

**Comment by glozow** [comment]
Thanks @hebasto, I've added your commit

**Review by hebasto — APPROVED** [review]
ACK 3d789657a8bf97089051348d472d7143a37ee988.

File: doc/man/bitcoin-cli.1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.3.
-.TH BITCOIN-CLI "1" "November 2023" "bitcoin-cli v26.0.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.1.
```
[the remark this answers is not in the document]
Comment (glozow) [reply doc/man/bitcoin-cli.1:1]: I suppose that's because fanquake and I use different machines

**Comment by glozow** [comment]
Added backport of #29195

**Review by hebasto — APPROVED** [review]
re-ACK 1e7fb270d36310efff6fc968f1c52291043d461b

**Comment by dergoegge** [comment]
Fuzz job times out because https://github.com/bitcoin/bitcoin/pull/28933 has not been back-ported

**Comment by glozow** [comment]
> (Up to you if you want to fix the fuzzing, I don't need it, I just wanted to mention that the fix is missing)

Seems better to leave it as is

**Review by fanquake — APPROVED** [review]
ACK 1e7fb270d36310efff6fc968f1c52291043d461b

**Pull request merged**